<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ActivateClass</class>
 <widget class="QWidget" name="ActivateClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>342</width>
    <height>199</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Activate</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>唯一ID</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="textEdit">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>岛主名字</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="textEdit_2">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="activateButtun">
     <property name="text">
      <string>激活</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="unActivateButtun">
     <property name="text">
      <string>注销</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Activate.qrc"/>
 </resources>
 <connections/>
</ui>
